_posts/2017-11-03-suds.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: SUDS
+date: 2017-11-2
+categories:
+  - projects
+description: Shower Use Detection System
+image: https://csh.rit.edu/~bigdata/SUDS.jpg
+image-sm: https://csh.rit.edu/~bigdata/SUDS.jpg
+author: Seth Gower
+author-image: https://avatars3.githubusercontent.com/u/19225089
+author-bio: First year Computer Engineering student
+author-email: seth@sethgower.com
+author-social:
+  github: https://github.com/SethGower
+  linkedin: https://www.linkedin.com/in/sethgower/
+---
+[SUDS](https://github.com/sethgower/suds), or the Shower Use Detection System, shows the status of the showers in the specific bathroom (and potentially the toilet stalls later on). When someone is in the shower and the door is locked, an LED on a panel in the hall next to the bathroom illuminates. In addition to the LEDs outside of the bathroom, I will be creating a web app to track live status updates remotely, and potentially collect statistics on when the showers/stalls are most used.
+
+SUDS is the first big hardware project that I have ever actually done, and the first time I have soldered my own board (outside of soldering headers onto MCUs and chips). The web app that I am going to build will be my first time using a MySQL database and creating a server-side backend that handles all of the input from the bathrooms.
+
+Originally, SUDS was not mine. It is a historical project that was started a few years ago, that instead of running on a Raspberry Pi, was a simple circuit driven by 2 AA batteries in the panel outside of the bathroom.
+
+As I mentioned, SUDS is driven by a Raspberry Pi in the ceiling. The RasPi outputs 5V (I realize that it is a bit much, will be lowering it during the rollout) which then goes to a limit switch on the door lock which when the lock is closed, it closes the circuit and illuminates the LED for that shower. The wiring diagrams can be found in the [GitHub page](https://github.com/sethgower/suds). On the same row on the proto-board as the anode of the LED, there is a pulldown resistor that leads to the GPIO pin corresponding to that shower.
```

Comment:
>As I mentioned,

You don't really have to add this part, IMO, maybe change that sentence to something like:

> SUDS is driven by a Raspiberry Pi in the bathroom ceiling, which outputs 5V...

> (I realize that is a bit much, ... )

Don't really think this parenthetical needs to be here. Maybe just mention your plans to change the circuitry setup instead.

This block generally has a run-on/comma-splice, as well.

> found in the Github page

:%s/in/on/g, IMO

>On the same row on the proto-board as the anode of the LED, ...

Maybe something more like:

>On the proto-board, on the same row as the anode of the LED, ...